docs/browser.md

```diff
@@ -0,0 +1,14 @@
+## jquery.flot.browser.js
+
+This plugin is used to make available some browser-related utility functions.
+
+### getPageXY
+Use getPageXY to obtain the viewable area of the page as set by the scroll bars.
+
+### getPixelRatio
+This function returns the current pixel ratio defined by the product of desktop
```

Comment:
You could include a link to this article: https://www.html5rocks.com/en/tutorials/canvas/hidpi/

Reply:
Done